password.py

```diff
@@ -1,49 +1,302 @@
+            print("Username not found!")
+            continue
+
+    account_dict_file.close()
 
     return
 
 #<---- SECOND MAJOR FUNCTION ---->
 # Login to existing account
 def login():
+    account_dict_file = open('account.txt','r')
 
+    attempts = 4
+    for line in account_dict_file:
+        word = re.split('\{|: |, |\}', line)
+        name = word[1]
+        username = name.replace("'","")
+        pass_w = word[6]
+        password = pass_w.replace("'","")
+
+    while True:
+        print("Enter your username: ")
+        user_input_username = input("")
+
+        if user_input_username == username:
+            while True:
+                print("Enter your password: ")
+                user_input_password = input("")
+                if user_input_password == password:
+                    password_manager()
+                    break
+                elif attempts == 0:
+                    print("You have exceeded max password attempts!")
+                    break
+                else:
+                    attempts -= 1
+                    print("Password not found!")
+                    print("You have "+ str(attempts) +" attempt(s) left.")
+            break
+
+        else:
+            print("Username not found!")
+            continue
+
+    account_dict_file.close()
+    
     return
+    
+#<---- THIRD MAJOR FUNCTION ---->
+# Manages and Rates your passwords.
+def password_manager():
+
+    if password_randomizer_query() == 'no':
+        while True:
+            password = input("Enter a Password:\n")
+            # Fail Case 1
+            if len(password) < 6:
+                print("Password has to be 6 characters or more.")
+                continue
+            # Fail Case 2
+            elif any(character.isdigit() for character in password) == False:
+                print("Password has to include at least one digit/number.")
+                continue
+            # Fail Case 3
+            elif bool(re.match("^[a-zA-Z0-9_]*$", password)) == True:
+                print("Password has to include at least one special character")
+                continue
+            else:
+                account_info_dict["Password"] = password
+                password_rating(password)
+                print("Would you like to save your password? (Type 'yes' or 'no'): ")
+                save_query = input("")
+                if save_query == 'yes':
+                    password_save()
+                    print("Password Saved.")
+                    break
+                elif 'no':
+                    continue
+                else:
+                    break
+    else:
+        while True:
+            print("Would you like to save your password?(Type 'yes' or 'no'): ")
+            save_query = input("")
+            if save_query == 'yes':
+                password_save()
+                print("Password Saved.")
+                break
+            elif 'no':
+                password_randomizer_query()
```

Comment:
This is supposed to rerun the 'password_manager' function again, not the 'password_randomizer_query' function.

Reply:
You can add commits that change things after you've posted a pull request. Just make the change, commit it, and push to the branch instead of to master.